packages/flutter_tools/lib/src/hot.dart

```diff
@@ -242,44 +244,68 @@ class HotRunner extends ResidentRunner {
     _devFS = null;
   }
 
-  Future<Null> _launchFromDevFS(ApplicationPackage package,
-                                String mainScript) async {
-    String entryPath = path.relative(mainScript, from: _devFSProjectRootPath);
-    String deviceEntryPath =
-        _devFS.baseUri.resolve(entryPath).toFilePath();
-    String devicePackagesPath =
-        _devFS.baseUri.resolve('.packages').toFilePath();
-    String deviceAssetsDirectoryPath =
-        _devFS.baseUri.resolve('build/flx').toFilePath();
+  Future<Null> _launchInView(String entryPath,
+                             String packagesPath,
+                             String assetsDirectoryPath) async {
     String viewId = await serviceProtocol.getFirstViewId();
-
     // When this completer completes the isolate is running.
     // TODO(johnmccutchan): Have the framework send an event after the first
     // frame is rendered and use that instead of 'runnable'.
     Completer<Null> completer = new Completer<Null>();
     StreamSubscription<Event> subscription =
-        serviceProtocol.onIsolateEvent.listen((Event event) {
-      if (event.kind == 'IsolateStart') {
-        printTrace('Isolate is spawned.');
-      } else if (event.kind == 'IsolateRunnable') {
-        printTrace('Isolate is runnable.');
-        completer.complete(null);
-      }
+       serviceProtocol.onIsolateEvent.listen((Event event) {
+     if (event.kind == 'IsolateStart') {
+       printTrace('Isolate is spawned.');
```

Comment:
This is existing code, but instead of calling `printTrace` for specific isolate events, you could call printTrace for any isolate event. It's a no-op if the user didn't start with `-v`, and if they did, the additional info could be useful.